cg/meta/upload/scout/rnafusion_config_builder.py

```diff
@@ -56,12 +60,20 @@ def _include_file(self, scout_key) -> None:
             self.get_file_from_hk(getattr(self.case_tags, scout_key)),
         )
 
+    def include_sample_alignment_file(self, config_sample: ScoutIndividual) -> None:
+        """Include the RNA sample alignment file."""
+        config_sample.rna_alignment_path = self.get_sample_file(
+            hk_tags=self.sample_tags.alignment_file, sample_id=config_sample.sample_id
+        )
+
+    def include_sample_files(self, config_sample: ScoutIndividual) -> None:
+        """Include all files that are used on RNA sample level in Scout"""
+        return None
```

Comment:
I don't understand the purpose of this function, why does it only return None?

Reply:
I changed the default behaviour of the parent class to return None, then we don't have to include this one 👍 

Thanks for the review @ChrOertlin ⭐